setup.py

```diff
@@ -6,6 +6,9 @@
       author="Ewen Cheslack-Postava",
       author_email='ewen@confluent.io',
       platforms=["any"], 
+      entry_points={
+        'console_scripts': ['ducktape=ducktape.__main__:main'],
```

Comment:
Ooops, I read that too fast. I was just wondering what the "right" way to make these types of commands work. Is putting a `__main__.py` in the top-level module the normal way to expose a command? I know that lets you do `python <module_name>`, but I wasn't sure if that was the common pattern for exposing things you want to run as just `<module_name>` (or really `<arbitary_binary_name>`).